integration_test/docker/docker-compose.yaml

```diff
@@ -101,7 +101,7 @@ services:
       - --logLevel=debug
 
   guardian-1:
-    image: alephium/guardiand:0.2.56
+    image: wormhole/guardiand:0.2.56
```

Comment:
We probably can't push these images to dockerhub, perhaps we should still use the alephium namespace?

Reply:
Fixed, thanks!